frontend/pages/api/auth/[...nextauth].ts

```diff
@@ -30,7 +30,7 @@ const options = {
 
       if (isSignIn) {
         const response = await fetch(
-          `http://localhost:1337/auth/${account.provider}/callback?access_token=${account?.accessToken}`
+          `${process.env.NEXT_PUBLIC_API_URL}/auth/${account.provider}/callback?access_token=${account?.accessToken}`
```

Comment:
Hey @ghoshnirmalya this might not work when you copy env.example having `NEXT_PUBLIC_API_URL=http://localhost:1337/graphql`

Just left a [PR](https://github.com/ghoshnirmalya/nextjs-strapi-boilerplate/pull/6) on this, because **/graphql/auth** won't work...